packageDependencies.plist

```diff
@@ -7,7 +7,7 @@
 	<key>awsCRTSwiftVersion</key>
 	<string>0.3.1</string>
 	<key>clientRuntimeBranch</key>
-	<string>main</string>
+	<string>jbe/acceptors</string>
```

Comment:
The smithy PR https://github.com/awslabs/smithy-swift/pull/483 must merge along with this one, and this must be reverted to `main` prior to this PR merging

Reply:
(Could also point this to `feat/waiters` if not merging the Waiters feature immediately after this)